lib/data_update_scripts/20200825095213_remove_orphaned_buffer_updates_by_article.rb

```diff
@@ -1,13 +1,7 @@
 module DataUpdateScripts
   class RemoveOrphanedBufferUpdatesByArticle
     def run
-      # Delete all BufferUpdates belonging to Articles that don't exist anymore
-      ActiveRecord::Base.connection.execute(
-        <<~SQL.squish,
-          DELETE FROM buffer_updates
-          WHERE article_id NOT IN (SELECT id FROM articles);
-        SQL
-      )
+      # We removed Buffer updates from the codebase, so this is now a no-op.
```

Comment:
Love that you went back and removed these so they wouldn't break for people cranking this up in the future! 🎉

Do we need to keep the files? I'm unfamiliar with the implementation we're using for data update scripts.

Reply:
Yeah, so often we do keep them around, but sometimes we just remove them. Good that you brought this up, I think I'm just going to nix them.